src/common/types/actions.types.ts

```diff
@@ -85,6 +87,11 @@ export interface UpdateCollateralTokenBalance {
   collateralTokenBalance: MonetaryAmount<Currency<CollateralUnit>, CollateralUnit>;
 }
 
+export interface UpdateGovernanceTokenBalance {
+  type: typeof UPDATE_GOVERNANCE_TOKEN_BALANCE;
+  governanceTokenBalance: MonetaryAmount<Currency<CurrencyUnit>, CurrencyUnit>;
```

Comment:
We should have `GovernanceUnit` available in the latest version of the lib on the master branch of this repo